src/en/reference-layer-yaml.md

```diff
@@ -0,0 +1,68 @@
+Title: Charm Layer.yaml Reference  
+
+# Layer.yaml
+
+Each layer used to build a charm can have a `layer.yaml` file. The top layer
+(the one actually invoked from the command line) must. These tell the generator
+what do, ranging from which base layers to include, to which interfaces. They
```

Comment:
to which interfaces...

did you miss something out or is that it? which base layers and interfaces to include?